js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties

```
@@ -3515,3 +3515,9 @@ oid4vciEnabled=Enable OID4VCI
 oid4vciEnabledHelp=Enable this option to allow the client to request verifiable credentials from Keycloak's OID4VCI credential endpoint.
 noAccessPolicies=No access policies
 noAccessPoliciesInstructions=There haven't been configured any access policies yet. Click the button below to configure the first policy.
+oid4vciAttributes=OID4VCI Attributes
```

I notice other tabs do not use the title case. Maybe follow the trend?
```suggestion
oid4vciAttributes=OID4VCI attributes
```